org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/manager/model/impl/Preset.java

```diff
@@ -40,20 +40,37 @@
 import org.openjdk.jmc.console.ext.agent.manager.model.IPreset;
 
 public class Preset implements IPreset {
+	private static final String DEFAULT_FILE_NAME = "newFile";
+	private static final String DEFAULT_CLASS_PREFIX = "__JFREvent";
+	private static final boolean DEFAULT_BOOLEAN_FIELD = true;
```

Comment:
Why default to `true`?

For safety reasons, `allowtostring` and `allowconverter` are actually opt-in features.

https://github.com/openjdk/jmc/blob/3eb5cb2b915fa3c93a91357034dadca9bd63d1f9/agent/src/main/java/org/openjdk/jmc/agent/impl/DefaultTransformRegistry.java#L276